Makefile

```diff
@@ -48,6 +48,6 @@ repl:
 		swift -F '$(TMPDIR)/SQLite.swift/Build/Products/Debug'
 
 sloc:
-	@zsh -c "grep -vE '^ *//|^$$' SQLite/*.{swift,h,c} | wc -l"
+	@zsh -c "grep -vE '^ *//|^$$' SQLite/*/*.{swift,h,m} | wc -l"
```

Comment:
Good catch! I don't think the depth is deeper than one directory here at the moment, but maybe `**/*` for future safety?